perfkitbenchmarker/scripts/object_storage_api_tests.py

```diff
@@ -100,6 +100,10 @@
                    'to start the operation. Only applies to the '
                    'MultiStreamRead and MultiStreamWrite scenarios.')
 
+flags.DEFINE_string('storage_class', None, 'The storage class to use for '
```

Comment:
Please rename this flag to "object_storage_class", and clarify in the text that this only applies to AWS because they support per-object storage class. For the other two providers, storage class is decided by the Bucket which is passed in as the --bucket parameter.